tshalvi/sonic-buildimage
# [Mellanox] Fix Mellanox-SN5600-V256 SKU for CMIS host management

#### Why I did it
On Mellanox platforms, certain configuration files related to the CMIS host management feature were not updated for the V256 SKU, leading to link issues on some ports.

##### Work item tracking
- Microsoft ADO **(number only)**:

#### How I did it
 updated two files for the V256 SKU: **media_settings.json** to include SerDes SI parameters for the relevant modules, and **sai_5600_256x200g.xml** to resolve a known host_tx_ready issue.

#### How to verify it
Enable the CMIS host management feature on the V256 SKU and verify that all ports successfully come up.



#### Which release branch to backport (provide reason below if selected)




#### Tested branch (Please provide the tested image version)




#### Description for the changelog




#### Link to config_db schema for YANG module changes


#### A picture of a cute animal (not mandatory but encouraged)
